test/BUILD.bazel

```diff
@@ -38,6 +38,28 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "backward",
+    alwayslink = True,
+    srcs = ["backward-stacktrace.cc"],
+    hdrs = ["backward-stacktrace.h"],
+    copts = copts(),
+    visibility = ["//visibility:public"],
+    deps = [
+        "@com_github_backward_cpp//:backward",
+        "@com_github_google_googletest//:gtest",
+    ],
+)
+
+eventuals_deps = [
```

Comment:
I'd inline this rather than have a variable for it.

I think of a shared deps list like this as a bit of an anti-pattern, since it can make it easy to have unnecessary dependencies in the shared list that are hard to spot or remove. From https://bazel.build/rules/tutorial-sharing-variables:

> Code duplication in BUILD files is usually fine. This can make the file more readable: each declaration can be read and understood without any context. This is important, not only for humans, but also for external tools. For example, a tool might be able to read and update BUILD files to add missing dependencies. Code refactoring and code reuse might prevent this kind of automated modification.